File: vuebugtrackerproject.client/src/enumConsts/ProjectPermission.ts
//Describes a role a user has with the project.
const ProjectPermission = {
  //Grants the user permission to view a project. Only used if the project is restricted.
  Viewer: 0,
  //Allows the user to add and modify bugs in a project.
  Editor: 1,
  //Indicates the user owns the project. This is only assigned when the user creates the project.
  Owner: 2
} as const;

export default ProjectPermission;

File: VueBugTrackerProject.Classes/Enums/ProjectPermission.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VueBugTrackerProject.Classes
{
    /// <summary>
    /// Describes a role a user can have in a project.
    /// </summary>
    public enum ProjectPermission
    {
        /// <summary>
        /// Grants the user permission to view a project. Only used if the project is restricted.
        /// </summary>
        Viewer,
        /// <summary>
        /// Allows the user to add and modify bugs in a project.
        /// </summary>
        Editor,
        /// <summary>
        /// Indicates the user owns the project. This is only assigned when the user creates the project.
        /// </summary>
        Owner

    }
}
